src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -16,17 +16,20 @@
 @Slf4j
 public class JdbcUserDao implements UserDao {
 
-    private final UserRowMapper userRowMapper = new UserRowMapper();
-    private final static String FIND_BY_EMAIL = "SELECT id, email, password, salt, iterations, role, status, discogs_user_name" +
-            " FROM public.users" +
-            " WHERE email=?";
-    private static final String INSERT = "INSERT INTO public.users" +
+    private static final RowMapper<User> ROW_MAPPER = new UserRowMapper();
+    private static final String FIND_BY_EMAIL = "SELECT id, email, password, salt, iterations, role, status, discogs_user_name" +
+            " FROM users" +
+            " WHERE email ILIKE ?";
+    private static final String FIND_BY_ID = "SELECT id, email, password, salt, iterations, role, status, discogs_user_name" +
+            " FROM users" +
+            " WHERE id=?";
+    private static final String INSERT = "INSERT INTO users" +
             " (email, password, salt, iterations, role, status, discogs_user_name)" +
-            " VALUES (?, ?, ?, ?, ?, ?, ?)";
-    private static final String DELETE = "DELETE FROM users WHERE email = ?";
-    private static final String UPDATE = "UPDATE public.users" +
+            " VALUES (?, ?, ?, ?, ?, ?, ?) RETURNING id";
```

Comment:
ON CONFLICT (email, discogs_user_name) DO NOTHING;
It works when email or discogs_user_name UNIQUE constraint is violated.
This will do for now but it's going to need changing when we fix the "can't have more than one empty discogsUsername in table, because all have to be unique" bug.